app/views/ads/new.html.haml

```diff
@@ -0,0 +1,5 @@
+%h1 New Ad
+= form_for @ad do |form|
+  = select_tag('category_id', options_for_select(@api.map { |item| [item["title"], item["id"]] }))
```

Comment:
try not to put any logic to views, first, you cannot really test it, and later on, it might be more difficult to debug it. so just leave here a variable `options_for_select(@api)`, and move mapping to the controller